{"uuid": "dcb0a7b3-0bbe-4c4c-8a9e-49a00da9b95c", "children": ["83930591-fe07-494b-8c89-5209d3f19d7f", "268f2232-039e-44c6-bac1-d7bdcb15fc8c", "a6760ea0-082e-49b1-b893-bbd6116d4181", "d564ab90-b67e-4c44-8553-c98a2f127ba8", "5bbaf02e-0962-44d8-bcdd-27aef140fbba", "6d5fa799-71df-4ee5-9509-3e88736a27d6", "701d2702-2375-41f3-8956-ec011f8296b7", "d90e4182-8a26-4dc1-a1b1-b4a370de43bc", "47476c4e-a9e6-4f7b-a8dd-613e1883b3dd", "e27e179c-d05e-46bd-b003-8d2a0ff343a6", "4e3389fa-35ec-4399-acb8-6d8ce2d8d91c", "addb5713-043c-447f-919b-7780cea1fe41", "43509e6b-19a9-450a-b568-c3ddb6c59e35", "a9bbc928-b946-46b7-8b75-a42ffb501fc9"], "befores": [{"name": "get_sponsors_page_content", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//label[contains(.,'logos')]/../../..//span[@class ='sc-dkPtRN cSldSA']\"}\n  (Session info: MicrosoftEdge=130.0.2849.80); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55ede8eff5be <unknown>\n#1 0x55ede89c69f6 <unknown>\n#2 0x55ede8a123ee <unknown>\n#3 0x55ede8a126c1 <unknown>\n#4 0x55ede8a56324 <unknown>\n#5 0x55ede8a35f7d <unknown>\n#6 0x55ede8a53efe <unknown>\n#7 0x55ede8a35cf3 <unknown>\n#8 0x55ede8a062de <unknown>\n#9 0x55ede8a06fbe <unknown>\n#10 0x55ede8ed4521 <unknown>\n#11 0x55ede8ed7791 <unknown>\n#12 0x55ede8ed720b <unknown>\n#13 0x55ede8ed7bc5 <unknown>\n#14 0x55ede8ec5ef2 <unknown>\n#15 0x55ede8ed7f4d <unknown>\n#16 0x55ede8eaf8c5 <unknown>\n#17 0x55ede8eefd18 <unknown>\n#18 0x55ede8eeff2f <unknown>\n#19 0x55ede8efe1bc <unknown>\n#20 0x7f6805894ac3 <unknown>\n\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 913, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QUBA_WEB/QUBA_WEB/Tests/test_SponsorsPage.py\", line 27, in get_sponsors_page_content\n    TestSponsorsPage.sponsors_page_content = admin_obj.get_sponsors_page_content()\n                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QUBA_WEB/QUBA_WEB/Pages/Admin.py\", line 358, in get_sponsors_page_content\n    \"logos\": self.get_sponsors_logos_filenames(),\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QUBA_WEB/QUBA_WEB/Pages/Admin.py\", line 320, in get_sponsors_logos_filenames\n    filenames.append(self.driver.find_element(*self.sponsors_logo_filename).text)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 741, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1732000921856, "stop": 1732000941676}, {"name": "get_sponsors_page_content", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//label[contains(.,'logos')]/../../..//span[@class ='sc-dkPtRN cSldSA']\"}\n  (Session info: MicrosoftEdge=130.0.2849.80); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55ede8eff5be <unknown>\n#1 0x55ede89c69f6 <unknown>\n#2 0x55ede8a123ee <unknown>\n#3 0x55ede8a126c1 <unknown>\n#4 0x55ede8a56324 <unknown>\n#5 0x55ede8a35f7d <unknown>\n#6 0x55ede8a53efe <unknown>\n#7 0x55ede8a35cf3 <unknown>\n#8 0x55ede8a062de <unknown>\n#9 0x55ede8a06fbe <unknown>\n#10 0x55ede8ed4521 <unknown>\n#11 0x55ede8ed7791 <unknown>\n#12 0x55ede8ed720b <unknown>\n#13 0x55ede8ed7bc5 <unknown>\n#14 0x55ede8ec5ef2 <unknown>\n#15 0x55ede8ed7f4d <unknown>\n#16 0x55ede8eaf8c5 <unknown>\n#17 0x55ede8eefd18 <unknown>\n#18 0x55ede8eeff2f <unknown>\n#19 0x55ede8efe1bc <unknown>\n#20 0x7f6805894ac3 <unknown>\n\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 913, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QUBA_WEB/QUBA_WEB/Tests/test_SponsorsPage.py\", line 27, in get_sponsors_page_content\n    TestSponsorsPage.sponsors_page_content = admin_obj.get_sponsors_page_content()\n                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QUBA_WEB/QUBA_WEB/Pages/Admin.py\", line 358, in get_sponsors_page_content\n    \"logos\": self.get_sponsors_logos_filenames(),\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QUBA_WEB/QUBA_WEB/Pages/Admin.py\", line 320, in get_sponsors_logos_filenames\n    filenames.append(self.driver.find_element(*self.sponsors_logo_filename).text)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 741, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1732000942708, "stop": 1732000956915}, {"name": "get_sponsors_page_content", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//label[contains(.,'logos')]/../../..//span[@class ='sc-dkPtRN cSldSA']\"}\n  (Session info: MicrosoftEdge=130.0.2849.80); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55ede8eff5be <unknown>\n#1 0x55ede89c69f6 <unknown>\n#2 0x55ede8a123ee <unknown>\n#3 0x55ede8a126c1 <unknown>\n#4 0x55ede8a56324 <unknown>\n#5 0x55ede8a35f7d <unknown>\n#6 0x55ede8a53efe <unknown>\n#7 0x55ede8a35cf3 <unknown>\n#8 0x55ede8a062de <unknown>\n#9 0x55ede8a06fbe <unknown>\n#10 0x55ede8ed4521 <unknown>\n#11 0x55ede8ed7791 <unknown>\n#12 0x55ede8ed720b <unknown>\n#13 0x55ede8ed7bc5 <unknown>\n#14 0x55ede8ec5ef2 <unknown>\n#15 0x55ede8ed7f4d <unknown>\n#16 0x55ede8eaf8c5 <unknown>\n#17 0x55ede8eefd18 <unknown>\n#18 0x55ede8eeff2f <unknown>\n#19 0x55ede8efe1bc <unknown>\n#20 0x7f6805894ac3 <unknown>\n\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 913, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QUBA_WEB/QUBA_WEB/Tests/test_SponsorsPage.py\", line 27, in get_sponsors_page_content\n    TestSponsorsPage.sponsors_page_content = admin_obj.get_sponsors_page_content()\n                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QUBA_WEB/QUBA_WEB/Pages/Admin.py\", line 358, in get_sponsors_page_content\n    \"logos\": self.get_sponsors_logos_filenames(),\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QUBA_WEB/QUBA_WEB/Pages/Admin.py\", line 320, in get_sponsors_logos_filenames\n    filenames.append(self.driver.find_element(*self.sponsors_logo_filename).text)\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 741, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1732000957946, "stop": 1732000971731}], "start": 1732000957946, "stop": 1732000971774}